docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.Re-enabling Keycloak breaks MTA
```

Comment:
@dymurray - if you are ok with this, i can merge ready to build to stage on Sunday
Thanks

Reply:
Looks great, thanks!